tests/python/pants_test/engine/legacy/test_bundle_integration.py

```diff
@@ -49,15 +48,9 @@ def test_bundle_directory(self):
     with self.bundled('directory') as bundle_dir:
       root = os.path.join(bundle_dir, 'a/b')
       self.assertTrue(os.path.isdir(root))
-      # NB: The behaviour of this test will change with the relevant deprecation
-      # in `pants.backend.jvm.tasks.bundle_create`, because the parent directory
-      # will not be symlinked.
-      deprecated_conditional(
-          lambda: os.path.isfile(os.path.join(root, 'file1.txt')),
-          '1.5.0.dev0',
-          'default recursive inclusion of files in directory',
-          'A non-recursive/literal glob should no longer include child paths.'
-      )
+      # NB: The behaviour of this test changed as scheduled in 1.5.0.dev0, because the
```

Comment:
Why leave this comment? It seems weird to reference the past. The reader won't know (and hopefully shouldn't care) what this refers to.

Reply:
My goal here would be that if someone opens a bug against the 1.5.0 dev series indicating that something is broken, someone would find this and determine (more easily) that the change was intentional... I know that these types of historical comments should eventually be deleted, but in the near term they seem helpful.

Reply:
Dunno, I feel like that's what git history is for. Let's at least put a "deprecation message" with a version on the comments, so that they get deleted in the find-and-replace sweep for a future version. And let's make sure they're reworded in past tense, so the reader doesn't go looking for options that no longer exist.